catalog/ui/catalog-ui-search/src/main/webapp/lib/jquery-ui-multiselect-widget/CHANGELOG

```diff
@@ -0,0 +1,171 @@
+jQuery UI MultiSelect Widget Changelog
```

Comment:
I don't really have a problem with it, but is it normal to commit all our libs? I would have expected to deploy them as an artifact instead.